app/grpc/gasestimation/gas_estimator.go

```diff
@@ -105,144 +88,120 @@ func (s *gasEstimatorServer) EstimateGasPriceAndUsage(ctx context.Context, reque
 	}, nil
 }
 
+// gasPriceEstimationThreshold the threshold of mempool transactions to
+// estimate the gas price.
```

Comment:
[nit] this comment doesn't explain what the threshold is used for. Can it be revised to say something about how min gas price is used if the block is less than this full?

Reply:
44c2934518157793282f70b8b1c3719fed64c2f4